### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by Jobjeuh**
Update ReadMe to reflect more modern standards and syntax.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,86 +16,79 @@ yarn add @scrumble-nl/react-quick-loader
...
 2. Passing through data manually (useful if you want one api call and multiple loaders)
 
-Both wil pass the data to the `data` property to all of their children. 
-The actual implementations are as followed:
+Both wil pass the data as a `data` prop to all of their children.
+The actual implementations are as follows:
 
 ### Url variant
-1. Import `QuickLoader`
+1. Import `QuickLoader` in the component where you want to create a toast
```
Comment (Luukdewaaier) [inline README.md:27]: Yo do no want to create a toast?

File: README.md [hunk 2]
```diff
@@ -16,86 +16,79 @@ yarn add @scrumble-nl/react-quick-loader
...
 | `color`      | string                                                                                 | *true*     | The color for the loader           |  |
-| `url`       | string                                                                                 | *true*    | The url used to retrieve data           | 
+| `url`       | string                                                                                 | *true*    | The url used to retrieve data           |
 | `type`       | 'blank', 'balls', 'bars', 'bubbles', 'cubes', 'cylon', 'spin', 'spinningBubbles', 'spokes'                                                                                 | *false*    | The loader type          | `bars` |
 | `errorCallback`      | (error: any) => void | *false*    | The custom callback for handling the error message                     | `undefined`
 
 ### Data variant
 The actual implementation is as follows:
-1. Import `QuickLoader`
+1. Import `QuickLoader` in the component where you want to create a toast
```
Comment (Luukdewaaier) [inline README.md:56]: same

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Quick-Loader
+# Quick-Toaster
```
Comment (Luukdewaaier) [inline README.md:1]: Can you also update the package to latest react version?

File: package.json [hunk 4]
```diff
@@ -34,29 +34,29 @@
     "lib/**/*"
   ],
   "peerDependencies": {
```
[the remark this answers is not in the document]
Comment (Jobjeuh) [reply package.json:36]: Ah right, forgot that

**Pull request merged**